src/fastoad/models/propulsion/fuel_propulsion/rubber_engine/rubber_engine.py

```diff
@@ -117,7 +124,14 @@ def compute_flight_points(self, flight_points: Union[FlightPoint, pd.DataFrame])
                 if name not in new_column_names:
                     flight_points.insert(len(flight_points.columns), name, value=np.nan)
 
-        flight_points.sfc = sfc
+        # SFC correction for NEO engines dependent on altitude.
+        f = interp1d(
```

Comment:
COdacy is not happy because `f` does not conform to snake_case. Maybe call this function `correction_factor_from_altitude` ?

Reply:
The name of the interpolation function has been changed to something more explicit.